src/main/java/no/nav/fpsak/nare/evaluation/BasicEvaluation.java

```diff
@@ -8,6 +8,7 @@
     private String ruleDescription;
     protected Resultat resultat;
     protected String reason;
+    private Object output;
```

Comment:
Ideelt sett bør vel denne være T og ikke object.. Men da må det fort deklareres i interfacet.

Reply:
Jeg lurte på å parametrisere både evaluation og rulereasonref - men er usikker på hvor hardt det treffer.
Mat for ny hovedversjon?

Reply:
Fjerner output og legger opp til bruk av RuleOutcome